system-tests/docker-test-env/nginx/testCredentials.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This certification expired on 2015.
Why are we declaring a HTTPS port for Nginx if are not using it in our origin configuration?

Reply:
For now this only servers as a configuration example for enabling TLS on NGINX. I couldn't get this to work without a certificate, and this old, self-signed test certificate appears to do just fine.

I could add this to the origins configuration if that helps?

Reply:
I have now added a secure origin.

Reply:
Maybe we should add toxyproxy config for that secure origin as well.